sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/health/KafkaWithoutHealthCheckIT.java

```diff
@@ -26,24 +29,38 @@
           // fresh kafka instance
           .withBrokerProperty("group.initial.rebalance.delay.ms", "0");
 
-  private static final DropwizardAppRule<KafkaTestConfiguration> DW =
-      new DropwizardAppRule<>(
-          KafkaWithoutHealthCheckTestApplication.class,
-          resourceFilePath("test-config-default.yml"),
-          config("kafka.brokers", KAFKA::getKafkaConnectString));
+  private static final LazyRule<DropwizardAppRule<KafkaTestConfiguration>> DW =
+      new LazyRule<>(
+          () ->
+              new DropwizardAppRule<>(
+                  KafkaWithoutHealthCheckTestApplication.class,
+                  resourceFilePath("test-config-default.yml"),
+                  config("kafka.brokers", KAFKA::getKafkaConnectString)));
```

Comment:
Do we still need the LazyRule here, if you are using a lambda for the config?

Reply:
solved :)